input/input_driver.c

```diff
@@ -39,16 +39,29 @@ bool input_driver_set_rumble(
          input_driver_state_t *driver_state, unsigned port, unsigned joy_idx, 
          enum retro_rumble_effect effect, uint16_t strength)
 {
-   const input_device_driver_t  *primary_joypad = driver_state->primary_joypad;
-   const input_device_driver_t      *sec_joypad = driver_state->secondary_joypad; /* NULL if none */
+   const input_device_driver_t  *primary_joypad;
+   const input_device_driver_t      *sec_joypad;
    bool rumble_state   = false;
 
+   if (!driver_state)
+   {
+      RARCH_ERR("Argument 'driver_state' is NULL in function 'input_driver_set_rumble'\n");
```

Comment:
I'm not a fan of the kind of warning spam this would cause.

Reply:
Understood. Fixed!